src/client.rs

```diff
@@ -53,13 +51,13 @@ impl RepliconRenetClientPlugin {
 
     fn set_connected(
         mut client: ResMut<RepliconClient>,
-        #[cfg(feature = "renet_transport")] transport: Res<NetcodeClientTransport>,
+        #[cfg(feature = "renet_netcode")] transport: Res<NetcodeClientTransport>,
     ) {
         // In renet only transport knows the ID.
         // TODO: Pending renet issue https://github.com/lucaspoffo/renet/issues/153
-        #[cfg(feature = "renet_transport")]
-        let client_id = Some(transport.client_id().to_replicon());
-        #[cfg(not(feature = "renet_transport"))]
+        #[cfg(feature = "renet_netcode")]
```

Comment:
Ah, I think we need to do the same for the steam feature :thinking:

Reply:
@matoous could you also take a look at this?

Reply:
Will take a look. Seems like `renet_steam` heavily relies on `steamworks::Client` so it might need further changes. Might need to read some more about how this works 👀